plat/allwinner/common/sunxi_bl31_setup.c

```diff
@@ -76,11 +77,11 @@ static void *sunxi_find_dtb(void)
 }
 
 void bl31_early_platform_setup2(u_register_t arg0, u_register_t arg1,
-				u_register_t arg2, u_register_t arg3)
+                                u_register_t arg2, u_register_t arg3)
```

Comment:
Sorry, but I don't like those white space changes. AFAIK using tabs and fill up the remaining "modulo" part with spaces is completely standard.  The issue with indentation in diffs is well known and people just learned to live with it. And I don't get the argument of using tabs != 8 spaces? Why would you do that?
We could have endless discussions about coding style, but the most important argument is for all to use the same one, as this simplifies review a lot.
And white space changes just for the sake of it are somewhat pointless anyway.

Reply:
Fair enough. I'll drop the commit.